hadoop-hdds/framework/src/main/resources/webapps/static/ozone.css

```diff
@@ -53,7 +53,36 @@ body {
 .sortorder.reverse:after {
     content: '\25bc';   // BLACK DOWN-POINTING TRIANGLE
 }
-
+.sorting:after {
```

Comment:
These changes, and a couple of related changes seem to be something that does not belong to this JIRA, and fixing something that is not mentioned here.
It is great to fix this thing, but for clarity I think anything that is related to how ordering elements are being rendered should be in a separate JIRA describing the problem, and then fixed in a separate PR, so later on we can track and understand what has happened and why, and we can more easily find the track records of a particular problem.

If I misunderstand it, and this change is required to deliver the DataNode ports from the backend to the frontend, and then show DataNode WebUI URLs properly, then please correct me, if I am right, please file a separate JIRA for this and add the improvement for the sorting UI and logic separately from adding the URLs.

Reply:
Removed Sorting related Data Rendering changes & will be tracked in a newly created  JIRA : HDDS-8465